pages/dashboard/v2/details/[id]/[studentEmail].js

```diff
@@ -51,10 +60,37 @@ export async function getServerSideProps(context) {
     return {};
   }
 
+  const certificationNumbers = await prisma.classroom.findUnique({
+    where: {
+      classroomId: context.params.id
+    },
+    select: {
+      fccCertifications: true
+    }
+  });
+
+  let superblockTitles = await getSuperblockTitlesInClassroomByIndex(
+    certificationNumbers.fccCertifications
+  );
+
+  let superblockURLS = await getDashedNamesURLs(
+    certificationNumbers.fccCertifications
+  );
+
+  let superBlockJsons = await getSuperBlockJsons(superblockURLS); // this is an array of urls
```

Comment:
Instead of a comment letting us know this is an array of URLs, could we make this variable more descriptive so that it describes what it is supposed to be?